libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -170,6 +170,17 @@ const AP_Param::GroupInfo AP_MotorsHeli_RSC::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("GOV_RANGE", 17, AP_MotorsHeli_RSC, _governor_range, AP_MOTORS_HELI_RSC_GOVERNOR_RANGE_DEFAULT),
 
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+    // @Param: AROT_SP
+    // @DisplayName: Autorotation Window Setpoint for External Governor
+    // @Description: The microsecond signal value sent to external governors to enable fast spool-up, when bailing out of an autorotation.  Set 0 to disable. If also using a tail rotor of type DDVP with external governor then this value must lie within the autorotation window of both governors.
+    // @Range: 0 1400
+    // @Units: PWM
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("AROT_SP", 18, AP_MotorsHeli_RSC, _ext_gov_arot_sp, 0),
+#endif
+
```

Comment:
@Gone4Dirt  remove your commit from today.  It appears that it is not passing all of the autotests.  I think I'm fine with letting this go in with the parameter being seen in all boards.  Please add a comment in the parameter description up front that says the parameter is only supported in SITL builds.  I will make a point to get the practice auto support in for all boards before 4.1 gets released.

Reply:
Ok, sounds good.  I have made those changes.